clustermesh-apiserver/main.go

```diff
@@ -7,16 +7,52 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/cilium/hive/cell"
 	"github.com/spf13/cobra"
 
 	"github.com/cilium/cilium/clustermesh-apiserver/clustermesh"
 	clustermeshdbg "github.com/cilium/cilium/clustermesh-apiserver/clustermesh-dbg"
 	"github.com/cilium/cilium/clustermesh-apiserver/etcdinit"
+	"github.com/cilium/cilium/clustermesh-apiserver/health"
 	"github.com/cilium/cilium/clustermesh-apiserver/kvstoremesh"
 	kvstoremeshdbg "github.com/cilium/cilium/clustermesh-apiserver/kvstoremesh-dbg"
+	cmmetrics "github.com/cilium/cilium/clustermesh-apiserver/metrics"
+	"github.com/cilium/cilium/clustermesh-apiserver/option"
+	"github.com/cilium/cilium/clustermesh-apiserver/syncstate"
 	"github.com/cilium/cilium/clustermesh-apiserver/version"
+	cmtypes "github.com/cilium/cilium/pkg/clustermesh/types"
 	"github.com/cilium/cilium/pkg/cmdref"
+	"github.com/cilium/cilium/pkg/controller"
+	"github.com/cilium/cilium/pkg/defaults"
+	"github.com/cilium/cilium/pkg/gops"
 	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/kvstore"
+	"github.com/cilium/cilium/pkg/kvstore/store"
+	"github.com/cilium/cilium/pkg/pprof"
+)
+
+var commonCell = cell.Module(
```

Comment:
Could you please move the definition of the common cell to a dedicated package, so that it is easier to extend in the future without polluting the main package?